.github/workflows/black.yaml

```diff
@@ -0,0 +1,11 @@
+name: Lint
+on: [push, pull_request]
+
+jobs:
+  lint:
```

Comment:
Nice addition! However, is this achieving what you want?

For example. from a recent run this step reported:
```
Run psf/black@stable
Run # Exists since using github.action_path + path to main script doesn't work because bash
reformatted todo_app/data/session_items.py
reformatted todo_app/flask_config.py
reformatted todo_app/tests/test_app.py
reformatted todo_app/tests/test_trello_items.py
reformatted todo_app/tests_e2e/test_selenium_e2e.py
All done! ✨ 🍰 ✨
[5](https://github.com/mpursell/DevOps-Course-Starter/runs/6453825014?check_suite_focus=true#step:3:6) files reformatted, 6 files left unchanged.
```

Which says it reformatted 6 files, but I don't think you're committing the changes or anything, so it just reformats them _again_ next time and keeps passing this step. I think you probably want to turn on [the `--check` argument to Black](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file) so that it fails the pipeline if it needs to reformat anything, or you _could_ look at automatically committing the linted changes, but I'd personally be a bit wary of that (if nothing else, it will potentially clog up your Git history strangely!)